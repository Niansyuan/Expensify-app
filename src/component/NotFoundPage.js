import React from "react";
import {Link} from 'react-router-dom'
const NotFoundPage=()=>(
    <div>
        404 <Link to='/'>Go back to HomePage</Link>
    </div>
)

export default NotFoundPage