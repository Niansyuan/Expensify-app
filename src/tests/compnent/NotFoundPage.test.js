//react-test-renderer
import React from "react";
import NotFoundPage from "../../component/NotFoundPage";
import { shallow } from "enzyme";
import "../setupTests";


test('should render NotFoundpage correctly',()=>{
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})