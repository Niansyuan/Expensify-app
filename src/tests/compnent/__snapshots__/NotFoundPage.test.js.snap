// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundpage correctly 1`] = `
<div>
  404 
  <Link
    to="/"
  >
    Go back to HomePage
  </Link>
</div>
`;
